.github/workflows/actions.yml

```diff
@@ -0,0 +1,48 @@
+name: Actions
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  
+  run-checks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - 'swift:5.10'
+    container:
+      image: ${{ matrix.image }}
+      
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 1
+    
+    - name: Check Unacceptable Language
+      uses: feather-actions/check-language@0.0.4
```

Comment:
Please use the appropriate workflows: https://github.com/swiftlang/github-workflows

Reply:
Gábor will do it later... :) He promised :D